core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

```diff
@@ -167,3 +173,64 @@ fun LpOutlinedTextFieldNumber(
         keyboardOptions = KeyboardOptions.Default.copy(keyboardType = KeyboardType.Number)
     )
 }
+
+/**
+ * [DropdownTextField] it's a textfield to show a list of items inside a box
+ *
```

Comment:
Please, add the param text to the documentation and the modifier to the documentation and function.